docs/introduction/Car-registry-tutorial.rst

```diff
@@ -2,73 +2,76 @@
-Accepting that cars will show up in sidechain in our example, we want to build an application that can store information that identifies a specific car, such as vehicle identification number, model, production year, color.
-We will also want car owners to prove their ownership of the cars without disclosing information about their identity. We also want users to sell and buy cars against ZEN coins. 
+The Car Registry app is an example of a sidechain that implements specific custom data and logic. The purpose of the application is to provide a simplified service that keeps records of existing cars and their owners. It is simplified as sidechain users will be able to register cars by merely paying a transaction fee. In contrast, in a real-world scenario, the ability to create a car will be bound by the presentation of a certificate signed by the Department of Motor Vehicles or analogous authority, or some other consensus mechanism that guarantees that the car exists in the real world and it’s owned by a user with a given public key.
+Accepting that cars will show up in the sidechain in our example, we want to build an application that has the following capabilitis:
 
+    1. It can store information that identifies a specific car, such as vehicle identification number (VIN), model, production year, color.
+    2. Allows car owners to be able to prove their ownership of the cars anonymously.
+    3. Require the use of ZEN for all transactions. 
+
+(Not sure that "Sidechain" should be capitalized as often as it is, but left it in case it's an internal formatting choice.)
 
 User stories:
 #############
 
 1
-**Q: I want to add my car to a Car Registry Sidechain.**
+**Q: I want to add my car to the Car Registry Sidechain.**
 
-*A:* Create a new Car Entry Box, which contains car identification information (Unique car identifier, VIN, manufacturer, model, year, registration number), and certificate. Proposition in this box is my public key in this Sidechain. When I create a box, Sidechain should check car identification information and certificate to be unique in this Sidechain.
+*A:* Create a new Car Entry Box, which contains car identification information (VIN, manufacturer, model, year, registration number), and certificate. Proposition in this box is my public key in this Sidechain. When I create a box, the Sidechain should verify car identification information and certificate are unique in this Sidechain.
```

Comment:
@zennermac 

*A:* Create a new Car Entry Box, which contains vehicle identification information (VIN, manufacturer, model, year, registration number), and a certificate. The proposition in this box is your public key in this sidechain. When you create a box, the sidechain should verify that the vehicle identification information and certificate are unique to this sidechain.